libkbfs/block_util_test.go

```diff
@@ -0,0 +1,86 @@
+package libkbfs
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestBlockUtilPutNewBlockSuccess(t *testing.T) {
+	mockCtrl, config, ctx := blockOpsInit(t)
```

Comment:
since we only use the mock block server, may as well have an init/shutdown function that returns just that